ush/python/pygfs/task/gfs_forecast.py

```diff
@@ -53,8 +53,10 @@ def initialize(self) -> None:
         self.gfs.stage_fix()
 
         # Stage the different tables
-        self.gfs.stage_tables(table='diag_table', target=os.path.join(self.task_config.DATA, 'diag_table.tmpl'))
-        self.gfs.stage_tables(table='field_table', target=os.path.join(self.task_config.DATA, 'field_table'))
+        self.gfs.stage_tables(table='diag_table', target=os.path.join(
+            self.task_config.DATA, 'diag_table.tmpl'))
+        self.gfs.stage_tables(table='field_table', target=os.path.join(
+            self.task_config.DATA, 'field_table'))
```

Comment:
I am curious about these and some more below.
Is something in your IDE that is forcing this change?
To me this makes the code less readable.